css/style.css

```diff
@@ -335,8 +336,101 @@ header {
   font-style: normal;
   font-weight: bold;
   font-size: 16px;
-  line-height: 15px;
+  line-height: 20px;
   align-self: center;
   text-decoration: none;
   padding: 0 12px;
 }
+
+.container-footer {
+  display: flex;
+  flex-flow: column nowrap;
+  row-gap: 28px;
+  border-radius: 88px 0 0 0;
+  background-color: #6070fe;
+  padding-bottom: 30px;
+  background-image: url(/images/bg-contact.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position-x: right;
+}
+
+.h2-contact-me {
+  color: #fff;
+  font-size: 40px;
+  font-weight: bold;
+  font-style: normal;
+  line-height: 50px;
+  text-align: center;
+  margin: 90px 0 0 0;
+}
+
+.p-contact-info {
+  color: #ebebff;
+  font-size: 20px;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 28px;
+  text-align: center;
+  padding: 10px 22px;
+}
+
+.contact {
+  margin: 2px 8px;
+  padding: 8px;
+}
+
+/* .container-footer {
```

Comment:
- [ ] [OPTIONAL]  It would nice if your removed all commented code from the file. This improves the overall code readability